raiden_installer/web.py

```diff
@@ -554,6 +555,27 @@ def get(self, configuration_file_name):
             }
         )
 
+    def post(self, configuration_file_name):
+        configuration_file = RaidenConfigurationFile.get_by_filename(configuration_file_name)
+        account = configuration_file.account
+        w3 = make_web3_provider(configuration_file.ethereum_client_rpc_endpoint, account)
+        ex_currency_amt = json_decode(self.request.body)
+        exchange = Exchange.get_by_name(ex_currency_amt['exchange'])(w3=w3)
+        currency = Erc20Token.find_by_ticker(ex_currency_amt['currency'], configuration_file.network)
+        token_amount = TokenAmount(ex_currency_amt['target_amount'], currency)
+        exchange_costs = exchange.calculate_transaction_costs(token_amount, account)
+        total_cost = exchange_costs["total"]
+        self.render_json(
+            {
+                "exchange": exchange.__class__.__name__.lower(),
```

Comment:
just a nitpick: exchange has a `name` property which returns the class name.

Reply:
Oops will make the change